pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -354,7 +365,10 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                     formatted='{pokemon} capture failed.. trying again!',
                     data={'pokemon': pokemon.name}
                 )
-                sleep(2)
+
+                # sleep according to flee_count and flee_duration config settings
+                sleep(choice([(x+1)*self.config.catchsim_flee_duration for x in range(self.config.catchsim_flee_count)]))
```

Comment:
Just some personal thoughts, you don’t have to change it to get my approval.
I feel like this process is a bit obscure and most users won’t understand what flee_duration and flee_count do. Looks like some kind of ```choice(range(start, stop, step))``` to me, but with the possibility to set the sleep to 0.
With : 
```
 choice(range(step, step*flee_count, step))
```